PWGHF/vertexingHF/AliHFSystErr.cxx

```diff
@@ -3082,6 +3090,65 @@ void AliHFSystErr::InitNonPromptDstoKKpi2017pp5TeVML() {
   return;
 }
 
+void AliHFSystErr::InitNonPromptDstoKKpi2018PbPb5TeV010ML(){
+  //
+  // Non-prompt Ds systematics in MB PbPb 5.02 TeV (2018) with BDT multi-class selections
+  // Ds --> K K pi
+  // Responsible: S. Politanò
+
+  AliInfo("Settings for Non-prompt Ds --> K K pi, MB PbPb collisions at 5.02 TeV - 2018 runs (LHC18qr) with BDT multi-class selections");
+  SetNameTitle("AliHFSystErr","SystErrNonPromptDstoKKpi2018PbPb010ML");
+
+  xbins[7] = [4., 6., 8., 12., 16., 24., 36.]
+
+  // Normalization
+  fNorm = new TH1F("fNorm","fNorm", 6, xbins);
```

Comment:
Optional: you could consider to add also lower pT bins (with `0.` values for the systematics) to avoid crashes in case you will try to extend the measurement to lower pT for the final result